pombola/core/fixtures/demodata.json

```diff
@@ -766,6 +766,54 @@
         "model": "core.organisationkind",
         "pk": 10
     },
+    {
+        "fields": {
+            "updated": "2013-08-01T01:21:27.682",
+            "name": "National Assembly",
```

Comment:
This should be replacing https://github.com/OpenUpSA/pombola/pull/87/files#diff-d4b94c1b6f0b2eaf1493cb867761ac7d8cbebef8b65507b2f088f647f99a188fL772 ? 

Please remove the orgs and org types that should be replaced - we don't want demodata to grow into an unmaintainable mess - it should be minimal

Reply:
delete these ones 
![Screenshot_2020-12-14_11-41-00](https://user-images.githubusercontent.com/235801/102065495-4e087a80-3e01-11eb-9ffa-d3c2f28a3fdc.png)